index.html

```diff
@@ -17,41 +17,30 @@
     })(window,document,'script','dataLayer','GTM-5N6WR7G');</script>
 
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/make-sense-ico.png" />
```

Comment:
Can we drop `%PUBLIC_URL%` part of the path now? 🤔 I'm not actively working in JS/TS anymore. Will it still work if we deploy it to the server?

Reply:
yes we can, the vite server will change it into static file.
**
![image](https://user-images.githubusercontent.com/28984914/180516001-0dfdcbe5-d4a1-46bb-b8ae-acc5d7235188.png)
**

all assets that are used inside public folder will be copied also.

Reply:
Greate! Taking a look at that pull request, I start to understand how much the tooling changed over the last few years. I think we will still deploy that brunch on some development instances. Just to make sure everything will work properly.